src/cmap/auth/mongodb_aws.ts

```diff
@@ -48,6 +51,7 @@ export class MongoDBAWS extends AuthProvider {
       throw new MongoMissingCredentialsError('AuthContext must provide credentials.');
     }
 
+    aws4 ??= loadAws4();
```

Comment:
Needs to be lazily loaded to pass the tests in: test/action/dependency.test.ts

Reply:
Which test fails?  I don't see any tests for aws4 in dependency.test.ts (should we add one?)

Reply:
(thought I replied)

It was the optional_require tests, but I had just forgotten to add .catch after converting from callback style